provider/resource_destination.go

```diff
@@ -32,6 +32,9 @@ func resourceDestination() *schema.Resource {
 		Update: resourceDestinationCreate,
 		Read:   resourceDestinationRead,
 		Delete: resourceDestinationDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceDestinationImportState,
```

Comment:
`StateContext` is the same as `State` but has an optional context. We do not use the context but `State` is depreciated in favor of `StateContext`.